front/app/containers/PlatformFooter/index.tsx

```diff
@@ -314,18 +317,19 @@ const PlatformFooter = ({
         </PagesNav>
 
         <Right>
-          <PoweredBy>
-            <PoweredByText>
-              <FormattedMessage {...messages.poweredBy} />
-            </PoweredByText>
-            <CitizenlabLink href="https://www.citizenlab.co/" target="_blank">
-              <CitizenLabLogo
-                name="citizenlab-footer-logo"
-                title="CitizenLab"
-              />
-            </CitizenlabLink>
-          </PoweredBy>
-
+          {removeCitizenlabBranding || (
```

Comment:
(continued from comment above) then you would have here 

```
{citizenlabBranding && (
            <PoweredBy>
              <PoweredByText>
                <FormattedMessage {...messages.poweredBy} />
              </PoweredByText>
              <CitizenlabLink href="https://www.citizenlab.co/" target="_blank">
                <CitizenLabLogo
                  name="citizenlab-footer-logo"
                  title="CitizenLab"
                />
              </CitizenlabLink>
            </PoweredBy>
          )}
```

Reply:
`{removeCitizenlabBranding || (
`
is pretty confusing.

Reply:
Then I'd rather do` !removeCitizenlabBranding &&`.

Imo more readable. Wdyt?

Reply:
> removeCitizenlabBranding ||

Is not a pattern we have in the front-end, so would stick with &&

Reply:
No problem for me. Done.